packages/components/tests/dummy/app/templates/components/gutters.hbs

```diff
@@ -0,0 +1,186 @@
+        >loose spacing scale in HDS</a>.)
+      </p>
+      <p>
+        <strong class="dummy-paragraph">Supported CSS units:</strong>
+        "px", "rem", "em", "%"<br />
+        (Negative values are not supported.)
+      </p>
+      <p class="dummy-paragraph">Default spacing size = "md"</p>
+    </dd>
+
+    <dt>direction</dt>
+    <dd>
+      <p>vertical, horizontal (Default, if any, to be determined)</p>
+    </dd>
+
+    <dt>TOOD: tag/inline/block</dt>
+    <dd>
+      <p>Needs more thought. (To prevent invalid markup, users would be able to choose between using a div tag (the
+        default) or a span tag.)</p>
+    </dd>
+
+    <dt>...attributes</dt>
+    <dd>
+      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+    </dd>
+  </dl>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="how-to-use">
+    <a href="#how-to-use" class="dummy-link-section">§</a>
+    How to use
+  </h3>
+
+  <h4 class="dummy-h4">Basic example with no custom spacing size set:</h4>
+  <p class="dummy-paragraph">(The default "md" spacing size will be rendered.)</p>
+  {{! prettier-ignore-start }}
+  <CodeBlock
+    @language="markup"
+    @code='
+      <Hds::Gutters @direction="horizontal">
+        <Hds::Button @text="Submit" type="submit" />
+        <Hds::Button @text="Cancel" @color="secondary" @href="https://hashicorp.com" />
+      </Hds::Gutters>
+    '
+  />
+  {{! prettier-ignore-end }}
+
+  <h4 class="dummy-h4">Example using a spacing value from the preset scale:</h4>
+  {{! prettier-ignore-start }}
+  <CodeBlock
+    @language="markup"
+    @code='
+      <Hds::Gutters @direction="vertical" @spacing="xxl">
+        <section>Child element #1</section>
+        <section>Child element #2</section>
+        <section>Child element #3</section>
+      </Hds::Gutters>
+    '
+  />
+  {{! prettier-ignore-end }}
+
+  <h4 class="dummy-h4">Example using a custom CSS unit spacing value:</h4>
+  {{! prettier-ignore-start }}
+  <CodeBlock
+    @language="markup"
+    @code='
+      <Hds::Gutters @direction="horizontal" @spacing=".5em">
+        <FlightIcon @name="alert-octagon" />
+        <span>Uh oh! Something happened.</span>
+      </Hds::Gutters>
+    '
+  />
+  {{! prettier-ignore-end }}
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="design-guidelines">
+    <a href="#design-guidelines" class="dummy-link-section">§</a>
+    Design guidelines
+  </h3>
+
+  {{! UNCOMMENT THIS BLOCK (once the link and/or the image are available) }}
+  <div class="dummy-design-guidelines">
+    <p class="dummy-paragraph">
+      <a
+        href="https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?node-id=2008%3A3"
+        target="_blank"
+        rel="noopener noreferrer"
+      >Figma UI Kit</a>
+    </p>
+  </div>
+</section>
+
+<section data-test-percy>
+  <h3 class="dummy-h3" id="showcase">
+    <a href="#showcase" class="dummy-link-section">§</a>
+    Showcase
+  </h3>
```

Comment:
Showcase examples could probably be improved if there are suggestions.

Reply:
No problem, once the ticket is in a more advanced state I can help with this